exec/java-exec/src/main/java/org/apache/drill/exec/planner/cost/DrillRelMdRowCount.java

```diff
@@ -53,7 +53,21 @@ public Double getRowCount(Aggregate rel, RelMetadataQuery mq) {
 
     if (groupKey.isEmpty()) {
       return 1.0;
-    } else {
+    } else if (rel instanceof AggPrelBase &&
+            ((AggPrelBase) rel).getOperatorPhase() == AggPrelBase.OperatorPhase.PHASE_1of2) {
+      // Phase 1 Aggregate would return rows in the range [NDV, input_rows]. Hence, use the
+      // existing estimate of 1/10 * input_rows
+        Double distinctRowCount = mq.getRowCount(rel.getInput()) / 10;
```

Comment:
Can we not get this default value from an existing Calcite utility ?  This will allow keeping in sync in the future if the ratio changes.

Reply:
Done